src/freenet/client/filter/HTMLFilter.java

```diff
@@ -2814,14 +2814,23 @@ private boolean isStandardLinkType(String token) {
 			String hreflang = getHashString(h, "hreflang");
 			String charset = null;
 			String maybecharset = null;
-            /* TODO: get the type from the filename. Currently we only
-             * have a filter for mp3, so this is the simplest possible
-             * solution.*/
-            String type = "audio/mpeg";
 
 			String src = getHashString(h, "src");
```

Comment:
Please extract this whole block into a method, I’m sure the containing method is large enough already… :)